src/js/tabs/configuration.js

```diff
@@ -196,6 +196,12 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
     load_config();
 
     function process_html() {
+        if (semver.gte(CONFIG.apiVersion, "1.41.0")) {
```

Comment:
There's more code dealing with `fpvCamAngleDegrees` on line 520. This should go there, to make maintenance easier.